lib/cooked_processor_mixin.rb

```diff
@@ -251,6 +253,33 @@ def add_broken_image_placeholder!(img)
     true
   end
 
+  def add_blocked_hotlinked_image_placeholder!(el)
+    el.name = "a"
+    el.set_attribute("href", el[PrettyText::BLOCKED_HOTLINKED_SRC_ATTR])
+    el.set_attribute("target", "_blank")
```

Comment:
Should it also set `rel="noopener noreferrer"`?

Reply:
Good call! Looks like it's automatically added by a later step in the CookedPostProcessor, so no need to add anything specific here. 

Added a safety-check in the specs to make sure that doesn't regress: 6ca0eb136e70a7a4484af2a732aa4f9aed160fa9